tests/unit/reports/test_utils.py

```diff
@@ -215,6 +218,333 @@ def test_get_column_plot_discrete_col(make_plot_mock):
     assert out == make_plot_mock.return_value
 
 
+def test_get_column_plot_invalid_sdtype():
+    """Test the ``get_column_plot`` method with an invalid sdtype.
+
+    Inputs:
+    - real column data
+    - synthetic column data
+    - column data type
+
+    Side Effects:
+    - A ValueError is raised.
+    """
+    # Setup
+    real_column = pd.Series([1, 2, 3, 4])
+    synthetic_column = pd.Series([1, 2, 4, 5])
+    sdtype = 'invalid'
+
+    # Run and assert
+    with pytest.raises(ValueError, match='sdtype of invalid not recognized'):
```

Comment:
This error message is a little confusing. Maybe change it to `sdtype 'invalid' not recognized` or `sdtype of type 'invalid' not recognized`, or something like that.